ts/screens/onboarding/PinScreen.tsx

```diff
@@ -305,17 +312,6 @@ class PinScreen extends React.Component<Props, State> {
         })
     );
   };
-
```

Comment:
you should also remove `showAbortOnboardingModal` from state usage, since it is useless

Reply:
updated within 9a3fb593dd5218cf6152e9d1d25915c60f4dee55